torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -123,13 +123,14 @@ std::shared_ptr<torch::jit::CompilationUnit> PythonRpcHandler::
   return jitCompilationUnit_;
 }
 
-py::object PythonRpcHandler::runPythonUdf(py::object&& pythonUdf) {
+py::object PythonRpcHandler::runPythonUdf(const py::object& pythonUdf) {
   PROFILE_GIL_SCOPED_ACQUIRE;
```

Comment:
Yes, making this change just because this PR removed the `movePythonUdf` method and let the UnpickledPythonCall take care of the py::object destruction. So that we no longer need a rvalue arg here. Hence, making this function consistent with other functions in PythonRpcHandler.